woocommerce/woocommerce
# [3.6] remove product list suggestions

### Changes proposed in this Pull Request:
Disable marketplace suggestions in products list.

Closes # .

### How to test the changes in this Pull Request:
1. Checkout this branch and get it set up as the WooCommerce plugin in your development environment.
2. Log in as a store administrator and view products list – should not see any suggestions.
3. View products & orders empty state, edit product metabox tab – these suggestions should be unaffected.
